HW1/P2/P2.txt

```diff
@@ -1 +1,20 @@
-# Your discussion here
+The strategy I used to more evenly distribute work over partitions was to
+randomize distribution of coordinate points. I found that using Spark's native
+repartition function gave a distribution of work centered closer to lower
+numbers of iteration, while first randomizing input coordinates with the
```

Comment:
Nice comparison! I wonder if there is more time overhead in repartitioning. And shouldn't repartition give essentially the same results as a shuffle?